### PR TITLE
Aesthetic Class Purge- Adept

### PR DESCRIPTION
**Pull request by rosebrallier**
Deleted Adept section

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -377,64 +377,6 @@ Skill Proficiencies:
...
-	also use any CS item as a melee tool that can hit at 5*INT + Lvl damage + 1x Plasma Burn,
-	-1 melee accuracy modifier (penalty when rolling against enemy Guard DC).
-	|
-	|
-	V
-	Items can be slightly more complex, up to .5x.5x.5m. Choose one of the 
-	following: CS Melee bash is extended to 5m, or deals 7*INT + Lvl damage + 1x Plasma Burn, or 
-	+1 melee accuracy modifier (bonus when rolling against enemy Guard DC).
-======================================
-
```
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:380]: *plays marching band music in celebration*

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -33,21 +33,21 @@ Prerequisites:
 == Acquired, Battle Ready ==
 Prerequisites: 
 
-*  Pilot or Adept
+*  Pilot
```
Comment (bleehu) [inline CharacterCreation/03_Feats.txt:36]: Good catch scrubbing feats!

File: CharacterCreation/06_01_Systemized_Skills.txt [hunk 3]
```diff
@@ -141,7 +141,6 @@ Example for (STRENGTH) Athletics check:
 
 ===== Class Skill Proficiencies =====
 
-Adept: Knowledge - Nanites, Weapons - General
 Assassin: Stealth, Weapons - Melee
```
Comment (bleehu) [inline CharacterCreation/06_01_Systemized_Skills.txt:144]: ...aaaaaand It's gone!!!

**Review by bleehu — APPROVED** [review]

**Review by Turtlelord26 — APPROVED** [review]
😃 🐢

**Pull request merged**